index.css

```diff
@@ -0,0 +1,617 @@
+  margin-left: 0;
+}
+
+.name {
+  font-family: cocogoose, sans-serif;
+  font-size: 12px;
+  text-align: left;
+  margin-top: 5px;
+  font-weight: bolder;
+  color: #3e3e3e;
+}
+
+.organize {
+  font-family: cocogoose, sans-serif;
+  font-style: italic;
+  font-size: 9px;
+  font-weight: bold;
+  text-align: left;
+}
+
+.speaker-description {
+  font-family: cocogoose, sans-serif;
+  word-break: keep-all;
+  text-align: left;
+  font-size: 9px;
+  line-height: 14px;
+}
+
+.grid-3 {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+}
+
+.partner-bg {
+  background-repeat: repeat;
+  background-color: rgb(62, 65, 67);
+}
+
+.text-color {
+  color: #e1e1e1;
+}
+
+.w-max-2 {
+  max-width: 52px;
+}
+
+.toggle-1 {
+  display: none !important;
+}
+
+.toggle-2 {
+  display: block;
+}
+
+.toggle-3 {
+  display: none !important;
+}
+
+#more {
+  width: 100%;
+}
+
+.about-text-lg-1 {
+  font-size: 28px;
+  font-weight: 900;
+  text-transform: uppercase;
+  width: 344px;
+}
+
+.about-text-lg-2 {
+  font-size: 28px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.grid-4 {
+  display: grid;
+  grid-template-columns: 100%;
+  gap: 1rem;
+}
+
+.partners {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  gap: 14px;
+}
+
+.bg-footer,
+.bg-footer-about {
+  background-color: #fff;
+}
+
+.grid-5 {
+  display: grid;
+  grid-template-columns: 50% 50%;
+  text-align: center;
+}
+
+.copyright {
+  font-family: 'cocogoose', sans-serif;
+  font-size: 9px;
+  font-weight: normal;
+  text-align: left;
+  color: #7a7a7a;
+  padding-top: 5px;
+}
+
+.w-max-1:hover {
+  box-shadow: 5px 10px 8px #888;
+  transform: scale(1.01);
+}
+
+a:hover {
+  color: rgb(190, 190, 190);
+}
+
+a:focus {
+  color: rgb(190, 190, 190);
+}
+
+@media screen and (min-width: 768px) {
+  #top-panel {
+    background-color: #3e3e3e;
+    width: 100%;
+    display: block;
+  }
+
+  .logo {
+    display: block !important;
+  }
+
+  .nav-ul {
+    display: flex !important;
+  }
+
+  .nav-text {
+    font-family: 'cocogoose', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 20px;
+    color: #3e3e3e;
+  }
+
+  .left-margin {
+    margin-left: 3rem !important;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  .grid {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    padding: 0 20px;
+  }
+
+  .program-section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  #btn {
+    padding-left: 0.8rem;
+    padding-right: 0.8rem;
+  }
+
+  .grid-2 {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  .toggle-1 {
+    display: block !important;
+  }
+
+  .toggle-2 {
+    display: none;
+  }
+
+  .toggle-3 {
+    display: block !important;
+  }
+
+  .text-card-1 {
+    /* font-family: cocogoose, sans-serif;
+    font-size: 12px;
+    font-weight: normal;
+    line-height: 16px; */
```

Comment:
Kindly, the previous reviewer told you to remove the commented codes, this will make your code look cleaner and tidier